homeassistant/components/hue/strings.json

```diff
@@ -104,6 +104,15 @@
           "unidirectional_incoming": "Unidirectional incoming"
         }
       }
+    },
+    "switch": {
```

Comment:
Where are you even referencing this translation key?

Reply:
It's a leftover actually. I wanted to translate "Automation: <name of the automation>" but that is not yet possible with the current translation system.